### PR TITLE
Umit

### PR DESCRIPTION
**Pull request by UmitErsahis**
can you check please?

### CODE REVIEW

File: src/test/java/Test1/Test1.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package Test1;
+
+public class Test1 {
+
+    //test1
```
Comment (Ozleblebici) [inline src/test/java/Test1/Test1.java]: u should run test2 here

**Review by halilarslan1999 — APPROVED** [review]
confirmed

File: src/test/java/Test1/Test2.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package Test1;
+
+public class Test2 {
+    //test2
```
Comment (Trojanowska) [inline src/test/java/Test1/Test2.java:4]: Are you sure this is good choice\?

File: src/test/java/Test1/Test1.java (hunk 1 above)
Comment (UmitErsahis) [reply src/test/java/Test1/Test1.java]: updated in [9399400](https://github.com/euroTech-sdet/Batch4_GitPractice/pull/5/commits/9399400f6a2a605de460de8e225ec0d6de0f3a51)

File: src/test/java/Test1/Test2.java (hunk 2 above)
Comment (UmitErsahis) [reply src/test/java/Test1/Test2.java:4]: it should be test2, Malgo

Comment (Ozleblebici) [reply src/test/java/Test1/Test2.java:4]: I agree with u  @UmitErsahis

**Review by Ozleblebici — APPROVED** [review]
All good, well done!

File: src/test/java/Test1/Test1.java [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package Test1;
+
+public class Test1 {
+
+    //test2
```
Comment (Kkmyrat) [inline src/test/java/Test1/Test1.java:5]: i think this test is false!

File: src/test/java/Test1/Test2.java (hunk 2 above)
Comment (Kkmyrat) [inline src/test/java/Test1/Test2.java:4]: this must be test1

**Pull request merged**